chat_exporter/construct/transcript.py

```diff
@@ -160,7 +160,7 @@ async def export_transcript(self, message_html: str, meta_data: str):
             ("MESSAGE_COUNT", str(len(self.messages))),
             ("MESSAGES", message_html, PARSE_MODE_NONE),
             ("META_DATA", meta_data_html, PARSE_MODE_NONE),
-            ("DATE_TIME", str(time_now)),
+            ("DATE_TIME", datetime.now(timezone).strftime("%b %d, %Y, %I:%M:%S %p (%Z)")),
```

Comment:
why not use time_now? it is already a string with respect to the 12h/24h setting of the user